pkg/kubectl-argo-rollouts/cmd/list/rollloutinfo.go

```diff
@@ -95,44 +96,3 @@ func (ri *rolloutInfo) String(timestamp, namespace bool) string {
 	}
 	return fmt.Sprintf(fmtString, args...)
 }
```

Comment:
Got rid of rolloutStauts in favor of the one in info package since there was duplication and inconsistencies in logic.